indigo_app/templates/indigo_api/_task_list_all.html

```diff
@@ -0,0 +1,55 @@
+{% load humanize indigo_app %}
+
+{% if tasks %}
+<table class="table table-striped table-bordered">
+  <thead>
+    <tr>
+      <th scope="col">Task</th>
+      <th scope="col">Place</th>
+      <th scope="col">Work/Document</th>
+    </tr>
+  </thead>
+  <tbody>
+  {% for task in tasks %}
+  {% if task.state != 'pending_review' %}
```

Comment:
You can leave this out, we'll do the filtering in Django's view.

Reply:
Removed